### PR TITLE
Update Project Ironsides to Dev Insights

### PR DESCRIPTION
**Pull request by jaholme**
## Summary of the pull request
Switch all references in code and resw files from "Project Ironsides to Dev Insights"

## PR checklist
- [x] Part of #3577 
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: src/Properties/launchsettings.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "DevHome (Unpackaged)": {
       "commandName": "Project"
     },
-    "PI (Package)": {
+    "DevInsights (Package)": {
       "commandName": "MsixPackage",
       "commandLineArgs": "--utilitylaunch DevHome.PI.Exe ",
```
Comment (krschau) [inline src/Properties/launchsettings.json:11]: Do you want to rename the exe?

File: tools/PI/DevHome.PI/Strings/en-us/Resources.resw [hunk 2]
```diff
@@ -138,8 +138,8 @@
     <comment>The tooltip for a button which toggles the orientation of the bar between horizontal and vertical</comment>
   </data>
   <data name="DissociateButton.ToolTipService.ToolTip" xml:space="preserve">
-    <value>Dissociate so that Project Ironsides is not tracking any app</value>
-    <comment>{Locked="Ironsides"} The tooltip for a button which causes Project Ironsides to no longer track any app</comment>
+    <value>Dissociate so that Dev Insights is not tracking any app</value>
+    <comment>{Locked="Ironsides"} The tooltip for a button which causes Dev Insights to no longer track any app</comment>
```
Comment (krschau) [inline tools/PI/DevHome.PI/Strings/en-us/Resources.resw:142]: Need to update all the locks.

File: src/Properties/launchsettings.json (hunk 1 above)
Comment (jaholme) [reply src/Properties/launchsettings.json:11]: happening in phases, first phase is tackling the resw files. Then we'll rename the source/classes which would include the exe.

File: tools/PI/DevHome.PI/Strings/en-us/Resources.resw (hunk 2 above)
Comment (jaholme) [reply tools/PI/DevHome.PI/Strings/en-us/Resources.resw]: Good call thanks.

File: tools/PI/DevHome.PI/Strings/en-us/Resources.resw [hunk 3]
```diff
@@ -654,16 +654,16 @@
     <comment>Tooltip for the system memory usage box in the bar</comment>
   </data>
   <data name="PIDisplayName" xml:space="preserve">
```
Comment (timkur) [inline tools/PI/DevHome.PI/Strings/en-us/Resources.resw:656]: >PI [](http://example.com/codeflow?start=14&length=2)

A little more messy, but these should probably be renamed DIDisplayName....

File: tools/PI/DevHome.PI/Strings/en-us/Resources.resw [hunk 4]
```diff
@@ -654,16 +654,16 @@
...
-    <value>Dev Home Project Ironsides</value>
-    <comment>{Locked}The description of the Project Ironsides application</comment>
+    <value>Dev Insights</value>
+    <comment>{Locked}The description of the Dev Insights application</comment>
   </data>
   <data name="PIStartupTaskDisplayName" xml:space="preserve">
-    <value>Hotkey for Dev Home Project Ironsides</value>
-    <comment>{Locked="Ironsides"} The description of the startup task for Dev Home Project Ironsides. This shows up in task manager/Windows Settings &gt; Startup Apps.</comment>
+    <value>Hotkey for Dev Insights</value>
+    <comment>The description of the startup task for Dev Insights. This shows up in task manager/Windows Settings &gt; Startup Apps.</comment>
```
Comment (krschau) [inline tools/PI/DevHome.PI/Strings/en-us/Resources.resw:666]: Sorry to be annoying but I think you need to add a lock back in here :) Looking down the changes, it seems when it's the full string it's locked but not when it's part of a string.

**Review by timkur — APPROVED** [review]
:shipit:

File: tools/PI/DevHome.PI/Strings/en-us/Resources.resw (hunk 3 above)
Comment (jaholme) [reply tools/PI/DevHome.PI/Strings/en-us/Resources.resw:656]: yeah will be in my next PR, going to change the namespace, directory, executable name, etc. but I anted to keep this one purely aesthetic so it wouldn't interrupt anyone's pending PRs.

File: tools/PI/DevHome.PI/Strings/en-us/Resources.resw (hunk 4 above)
Comment (jaholme) [reply tools/PI/DevHome.PI/Strings/en-us/Resources.resw:666]: I'll reach out to branding to make sure we should be locking the entire name, I could see it being beneficial if "Dev Insights" was translated.

**Review by dkbennett — APPROVED** [review]

File: settings/DevHome.Settings/Strings/en-us/Resources.resw [hunk 5]
```diff
@@ -554,12 +554,12 @@
     <comment>Inline description of the Quiet background processes experimental feature on the 'Settings -&gt; Experiments' page where you enable it.</comment>
   </data>
   <data name="ProjectIronsidesExperiment_Name" xml:space="preserve">
-    <value>Project Ironsides</value>
-    <comment>Name of experimental feature 'Project Ironsides' on the 'Settings -&gt; Experiments' page where you enable it.</comment>
+    <value>Dev Insights</value>
```
[the remark this answers is not in the document]
Comment (jaholme) [reply settings/DevHome.Settings/Strings/en-us/Resources.resw:557]: following up with branding on their recommendations. I will send an updated PR once they get back to me.

**Pull request merged**